examples/tensorflow-resnet/README.md

```diff
@@ -0,0 +1,187 @@
+# Introduction
+
+This is a Gravity Cluster Image example using the TensorFlow ResNet (Deep Residual Network) server helm chart (2.0.13) that serves a machine learning (ML) model.  The default model is 
```

Comment:
`ResNet` has already been explained above so I think you can drop all further clarifications and use either term. Ditto for machine learning

Reply:
Since someone may be coming directly to the example I'd rather leave the explanation in.